.github/workflows/release.yml

```diff
@@ -3,7 +3,7 @@ name: Release
 on:
   push:
     tags:
-      - '*'
+      - "*"
```

Comment:
Thank you for formatting.  What tool did you use to format?  In another team we now use `yq .` to format workflow files.  The conventions with that look similar to this.  This formats the yaml but does not do any schema checking.
```
git ls-files .github/workflows/ | grep -E '[.]ya?ml$' | while read -r line ; do cat <<<"$(yq . "$line")" > "$line" ; done
```

Reply:
I'm honestly not sure. It's just VS Code's default YAML formatting, as far as I can tell.